pliny contains an error just when instlling it

**Describe the bug**
the file tsconfig.json on line 2 gives the error File '@pliny/config/tsconfig.base.json' not found.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to node modules
2. check pliny/tsconfig.json
ps : same error on tsup : Cannot find module 'tsup' or its corresponding type declarations.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
![image](https://github.com/timlrx/pliny/assets/52544684/7317c3bf-3567-4696-850d-07dc27b5b07f)
![image](https://github.com/timlrx/pliny/assets/52544684/99bb65e1-efe2-4822-b40f-28f933c75605)


**System Info (if dev / build issue):**
 - OS: windows 11 
 - Node.js v18.17.1
 -[ Npm version](npm: '10.8.1',)

on opera gx / google 

**Additional context**

--- a/.changeset/calm-islands-promise.md
+++ b/.changeset/calm-islands-promise.md
@@ -1,0 +1,5 @@
+---
+'pliny': patch
+---
+
+include only relevant files in dist

--- a/packages/pliny/package.json
+++ b/packages/pliny/package.json
@@ -15,7 +15,18 @@
     "./utils/*": "./utils/*"
   },
   "files": [
-    "**"
+    "analytics",
+    "comments",
+    "mdx-plugins",
+    "newsletter",
+    "search",
+    "ui",
+    "utils",
+    "mdx-components.js",
+    "mdx-components.d.ts",
+    "config.js",
+    "config.d.ts",
+    "chunk*.js"
   ],
   "scripts": {
     "dev": "tsup && cp -fR dist/* ./ && tsup --config tsup.ui.config.ts && rimraf dist && node add-use-client.mjs",

--- a/turbo.json
+++ b/turbo.json
@@ -1,6 +1,6 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "pipeline": {
+  "tasks": {
     "build": {
       "dependsOn": [
         "^build"

